tests/unit/storages/test_request_queue.py

```diff
@@ -62,6 +62,39 @@ async def test_consistency_accross_two_clients() -> None:
         await rq_by_id.drop()
 
 
+async def test_deduplication_of_requests_with_custom_unique_key() -> None:
+    await Request.from_url('https://apify.com', unique_key='apify', always_enqueue=True)
```

Comment:
```
ValueError: `always_enqueue` cannot be used with a custom `unique_key`
```

We probably want to fix this, but adding a new test for checking this would also be great, something like this:

```python
with pytest.raises(ValueError, match=' always_enqueue cannot be used with a custom unique_key'):
    await Request.from_url('https://apify.com', unique_key='apify', always_enqueue=True)
```